### DISCUSSION TITLE
How to use relative image paths in Markdown files?

### DISCUSSION THREAD

**Discussion by barcia** [11ty/eleventy#1845] [Q&A]
I want to save my markdown files and images in same folder, and use a different permalink file structure.

Example:

```
content/
  2021/
    01/
      my_post/
        index.md
        image.jpg
```

And my output configuration is something like that:
```
blog/
      my_post/
        index.html
```

I want to use relative path for images in Markdown files, and have the images copied to the output folder.

¿Is possible that with Eleventy?

**Comment by nhoizey** [comment]
I tried to do that when I started using Eleventy, but failed, so I moved to have the same folders structure in source and destination.

Images can then be managed with a "simple" `eleventyConfig.addPassthroughCopy('src/**/*.{jpg,jpeg,png,gif}'));`

Example:
https://github.com/nhoizey/nicolas-hoizey.com/tree/main/src/articles/2020/11/02/enhancing-archives-navigation-step-2
